news/2024/2024-04-09-skinning-contest-unchained.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "Skinning Contest: Unchained"
+date: 2024-04-09 15:00:00 +0000
+---
+
+Unshackle your creativity and showcase it to its fullest by participating in the newest skinning contest!
```

Comment:
Reads pretty weirdly, on first read my question is "showcase what?". Maybe just me.